scripts/zones/Tavnazian_Safehold/npcs/Despachiaire.lua

```diff
@@ -5,6 +5,7 @@
 -----------------------------------
 require("scripts/globals/missions")
 -----------------------------------
+-- TO DO: Quest giver for "X Marks the Spot" which requires COP 2-5 Ancient Vows completed
```

Comment:
We tend to search on `TODO` when we leave things like this in the code, so please change this to that. 

I think this is also a good opportunity to make an issue and put the issue number in with the TODO:
```
If you agree with staff and/or your reviewers that some work in your pull request can
 be left as "to-do", make new issues on GitHub for your new TODO items and put the 
ID alongside the comment. The comment should also be sufficiently descriptive of what
 the missing work is, and why it was left out. eg. // TODO: A Boy's Dream - PLD AF Quest 2 - 
Cannot be completed until fishing is implemented (GitHub Issue #12345)
```

Reply:
Done. See #1481
I added all missing quests and the issue reference.